src/features/wallet/helpers/shortenAddress.ts

```diff
@@ -0,0 +1,7 @@
+// Shorten ETH address to 0xaa...bbbb
+export default function shortenAddress(address: string) {
+  const leftSide = address.substr(0, 4);
```

Comment:
just a nit, but I think most dapps show the first 6 digits (0x included)

Reply:
Tom requested 4 chars -- super easy fix if we want to change it though!